tasks/section_6/cis_6.1.x.yml

```diff
@@ -212,8 +212,7 @@
   - name: "SCORED | 6.1.12 | AUDIT | Ensure no ungrouped files or directories exist | Displaying all ungrouped files or directories"
     debug:
       msg: "Manual intervention is required -- missing group on items in {{ item.item.mount }}: {{ item.stdout_lines | join(', ') }}"
-    changed_when:
-    - rhel7stig_audit_complex
+        #changed_when: - rhel7stig_audit_complex
```

Comment:
I couldn't find `rhel7stig_audit_complex` anywhere so I just commented this out.